### PR TITLE
Update the Kubernetes token docs to reflect the platform hub self-serve features

### PR DESCRIPTION
**Pull request by tasharnvb**

### CODE REVIEW

File: how-to-docs/README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ This folder contains a collection of how-to guides for both Developers and DevOp
   - [Generating Certs](ca_tls.md)
   - [Importing local Certs](local_certs.md)
   - [Creating namespaces in UK cluster](namespace-how-to.md)
-  - [How to get a Kubernetes token for the UK cluster](kubernetes-token.md)
+  - [Getting a Kubernetes token for the UK cluster](kubernetes-token.md)
```
Comment (KashifSaadat) [inline how-to-docs/README.md]: Nitpick: keep the case the same for `Token`, as it's referenced in other places

File: how-to-docs/kubernetes-robot-token.md [hunk 2]
```diff
@@ -1,13 +1,26 @@
-# How to get a Kubernetes Robot Token for the UK cluster
+# How to get a Kubernetes Robot Token for a service in the UK cluster
```
Comment (KashifSaadat) [inline how-to-docs/kubernetes-robot-token.md]: Nitpick: Maybe keep the header names in sync with the links in README.md

File: how-to-docs/kubernetes-robot-token.md [hunk 3]
```diff
@@ -1,13 +1,26 @@
...
+
+## Project Admins (Creating a robot token)
+
+1. Log into the [Platform Hub][platform hub link].
+
+2. Go to the [Projects][project list] section and find your project. Click on the **Services** tab and find the service that requires a robot token.
+
+3. Go to the **Kube Robot Tokens** tab and click the **Create a Kubernetes robot token for this service** button.
+
+4. Select the required cluster, RBAC groups, robot name and description for the robot token and click **Create**.
```
Comment (KashifSaadat) [inline how-to-docs/kubernetes-robot-token.md]: "What's my required RBAC group?"

Maybe we need to put some explanation around each of the fields here, and an example to refer to.

File: how-to-docs/kubernetes-token.md [hunk 4]
```diff
@@ -1,15 +1,23 @@
 # How to get a Kubernetes token for the UK cluster
```
Comment (KashifSaadat) [inline how-to-docs/kubernetes-token.md]: Rename to `kubernetes-user-token.md` to keep similar naming standard with the robots file.

File: how-to-docs/kubernetes-token.md [hunk 5]
```diff
@@ -1,15 +1,23 @@
...
 
-Log into the [platform hub][platform hub link].
+## Users
 
-Select **Support Requests** under **Help & Support** from the sidebar.
+1. Talk to your project admin and ask them to generate a user token for you.
```
Comment (KashifSaadat) [inline how-to-docs/kubernetes-token.md]: How do I know who is an admin for the project? Maybe give them some steps to get to the relevant project page and see who has the "Admin" tag.

File: how-to-docs/kubernetes-robot-token.md (hunk 3 above)
Comment (tasharnvb) [reply how-to-docs/kubernetes-robot-token.md]: Is there an example we could point to? or should we just make one up?

Comment (KashifSaadat) [reply how-to-docs/kubernetes-robot-token.md]: I'd say just make one up.

**Comment by tasharnvb** [comment]
I don't know whether it would be better to have the RBAC group explanation in a separate file and link it in both of the token docs or just have it juts written into both files. Thoughts?

File: how-to-docs/README.md [hunk 6]
```diff
@@ -12,8 +12,8 @@ This folder contains a collection of how-to guides for both Developers and DevOp
   - [Generating Certs](ca_tls.md)
   - [Importing local Certs](local_certs.md)
   - [Creating namespaces in UK cluster](namespace-how-to.md)
-  - [How to get a Kubernetes token for the UK cluster](kubernetes-token.md)
+  - [Getting a Kubernetes Token for the UK cluster](kubernetes-token.md)
```
Comment (KashifSaadat) [inline how-to-docs/README.md]: File was renamed so needs updating here (sorry) :)

**Pull request merged**